security/pkg/pki/util/crypto.go

```diff
@@ -61,14 +62,15 @@ func ParsePemEncodedCertificateChain(certBytes []byte) ([]*x509.Certificate, err
 		}
 		cert, err := x509.ParseCertificate(cb.Bytes)
 		if err != nil {
-			return nil, fmt.Errorf("failed to parse X.509 certificate")
+			return nil, nil, fmt.Errorf("failed to parse X.509 certificate")
 		}
 		certs = append(certs, cert)
+		rootCertBytes = certBytes
```

Comment:
This will make rootCertBytes as empty.   you might get root cert from the certs directly.

Reply:
@irisdingbj @hzxuzhonghu This issue has been fixed, PTAL, thanks!